drivers/filesystems/fastfat/misc.c

```diff
@@ -467,34 +490,72 @@ VfatCheckForDismount(
     /* If we were to delete, delete volume */
     if (Delete)
     {
-        PVPB DelVpb;
+        LARGE_INTEGER Zero = {{0,0}};
+
+        /* We are uninitializing, the VCB cannot be used anymore */
+        ClearFlag(DeviceExt->Flags, VCB_GOOD);
+
+        /* Close internal opens */
+        if (DeviceExt->RootFcb)
+        {
+#if 0
+            CcUninitializeCacheMap(DeviceExt->RootFcb->FileObject,
+                                   &Zero,
+                                   NULL);
+            ObDereferenceObject(DeviceExt->RootFcb->FileObject);
+#endif
+            vfatDestroyFCB(DeviceExt->RootFcb);
+            DeviceExt->RootFcb = NULL;
+        }
+        if (DeviceExt->VolumeFcb)
+        {
+#if 0
```

Comment:
Why commented out?

Reply:
That's something I'm not completely sure about it; a similar uninitialization seems to be performed in fastfat_new (in FatTearDownVcb() ) but I would hear about any advice.